qwandaq/src/main/java/life/genny/qwandaq/datatype/capability/core/node/CapabilityMode.java

```diff
@@ -18,7 +17,7 @@ public enum CapabilityMode {
      * Cached map to speed up identifier lookup
      * 
      */
-    private static Map<Character, CapabilityMode> idMap = new HashMap<>(values().length);
+    private static final Map<Character, CapabilityMode> idMap = new HashMap<>(values().length);
```

Comment:
we picked this up going over the capabilities system earlier today @jasperRob, should be final